### PR TITLE
 added eligible_payment_methods and eligible_payment_count

### PR DESCRIPTION
**Pull request by weihou**

### CODE REVIEW

File: src/component.js [hunk 1]
```diff
@@ -91,10 +91,16 @@ export let HostedFields = {
     // toodoo - revert change below when config is being passed correctly
     let configuration = (typeof __hosted_fields__ !== 'undefined') ? __hosted_fields__.serverConfig : TESTING_CONFIGURATION;
     configuration.assetsUrl = TESTING_CONFIGURATION.assetsUrl;
-    if (!configuration.card && configuration.paypalMerchantConfiguration && configuration.paypalMerchantConfiguration.creditCard) {
-      configuration.card = configuration.paypalMerchantConfiguration.creditCard;
-    } else {
-      // configuration.card = TESTING_CONFIGURATION.card;
+
+    let cardVendors = configuration.fundingEligibility && configuration.fundingEligibility.card && configuration.fundingEligibility.card.vendors;
+    let eligibleCards = [];
```
Comment (crookedneighbor) [inline src/component.js]: how about:

```js
let eligibleCards = Object.keys(cardVendors).filter(key => cardVendors[key].eligible);
```

**Comment by crookedneighbor** [comment]
FYI, something changed upstream a while ago that makes these builds fail. 😢

**Comment by bluepnume** [comment]
I'm a little concerned about exposing card types. Right now the cards returned are just using SPB's internal logic, and only cards renderable by SPB are returned. So it doesn't necessarily map to the cards accepted by hosted fields.

**Comment by bluepnume** [comment]
I fixed the tests in master btw.

**Comment by weihou** [comment]
I thought we have made the changes to expose the eligible card types to the users so that they won't enter the card info of an unacceptable card.

**Comment by weihou** [comment]
@bluepnume what causes this error? https://travis-ci.com/paypal/payment-sdk-hosted-fields-component/builds/99492692

**Comment by bluepnume** [comment]
Can you rebase against master

**Pull request merged**